include/triton/Tools/Sys/GetEnv.hpp

```diff
@@ -30,6 +30,10 @@ inline const std::set<std::string> CACHE_INVALIDATING_ENV_VARS = {
     "TRITON_LLVM_DEBUG_ONLY",
     "USE_TTGIR_LOC",
     "NVPTX_ENABLE_DUMP",
+    "SWP_FIRST_DOT",
+    "PEEL_EPILOGUE",
+    "LOAD_DIFFERENT_STAGE",
+    "FIRST_LOAD_OF_USE",
```

Comment:
Should these get a documentation line in the README.md?

Reply:
Yes. I will add them to README.md. We may not need all of these env variables, maybe we should always set loads to different stages, see this comment.

```
  // TODO pawel: we could do more fine-grained allocation here and
  // allocate only the number of buffers that specific loads need.
```